.github/workflows/test.yml

```diff
@@ -452,6 +452,7 @@ jobs:
       KIND_NUM_NODES_PER_ZONE: "${{ matrix.num-nodes-per-zone }}"
       OVN_DISABLE_FORWARDING: "${{ matrix.forwarding == 'disable-forwarding' }}"
       USE_HELM: "${{ matrix.target == 'control-plane-helm' || matrix.target == 'multi-homing-helm' }}"
+      OVN_ENABLE_DNSNAMERESOLVER: "${{ matrix.target == 'control-plane' && (( matrix.ha == 'HA' && matrix.gateway-mode == 'shared' && matrix.ipfamily == 'IPv6' && matrix.second-bridge == '1br' ) || ( matrix.ha == 'noHA' && matrix.gateway-mode == 'local' && matrix.ipfamily == 'IPv4' && matrix.second-bridge == '1br' ) || ( matrix.ha == 'noHA' && matrix.gateway-mode == 'shared' && matrix.ipfamily == 'IPv6' && matrix.second-bridge == '2br' )) }}"
```

Comment:
I think it will a bit more obvious if you use "enable-dns-name-resolver": "true" flag in the matrix and here just turn it on when it's true